src/lamplib/src/genny/curator.py

```diff
@@ -213,11 +213,18 @@ def __init__(
         if self._linux_distro in ("amazon2", "rhel8", "rhel62"):
             self._curator_distro = "rhel70"
 
+        if self._linux_distro == "amazon2arm":
+            self._curator_distro = "arm"
+
     def _get_url(self):
+        # The pinned curator version was before arm support was added. Use a newer version for Arm
+        version = CuratorDownloader.CURATOR_VERSION
+        if self._curator_distro == "arm":
+            version = "965d53845fd1987ddbf04a937ff625f3c243dee3"
```

Comment:
Can there be a `CuratorDownloader.ARM_CURATOR_VERSION` constant so the version numbers can be updated in one place?

Also consider making it a dict mapping curator distro to version.